src/SFA.DAS.FindApprenticeshipTraining.UnitTests/Application/Shortlist/Commands/WhenDeletingShortlistItemForUser.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.FindApprenticeshipTraining.Application.Shortlist.Commands.DeleteShortlistForUser;
+using SFA.DAS.FindApprenticeshipTraining.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.FindApprenticeshipTraining.UnitTests.Application.Shortlist.Commands
+{
+    public class WhenDeletingShortlistItemForUser
+    {
+        [Test, MoqAutoData]
+        public async Task Then_Creates_The_Shortlist_From_The_Request_Calling_CourseDelivery_Api(
```

Comment:
This test is confusingly named, this is a deletion test, not a creation test, right?

Reply:
Updated 👍🏻